### DISCUSSION TITLE
9.0.1 (SDK rework, "Enable OC" setting)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#459] [Announcements]
###  Changes into this release:

`Light control SDK`:
- Code rework for APIv2,3,6,9 (older systems and monitors). All light control tools smaller now.
- Code rework for lights save for APIv2,3 (older systems). Hope it will work correctly now (i don't have v2 hardware to test now).

`Fan control SDK`:
- Bug fixed: TCC control detected as present for some systems doesn't support it.

`All fan control GUI tools`:
- New: "Enable OC" setting added to settings (top menu for Fan GUI, Settings tab for FX GUI). It's disabled by default, you need to enable it if you plan to use OC functions. It's useful for systems with buggy OC support.

`All CLI tools`:
- Documentation updated to reflect current state.

`Known issues`:
- GUI manuals and guides a bit outdated, update in progress.

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/9.0.1'>9.0.1 (SDK rework, "Enable OC" setting)</a>.</em>

**Comment by Und3r22** [comment]
thanks for the app! 
I only use it for fans, my G15 i5 13th doesn't have lights.
I would like to know about power mode, in the menu I have the options "g-mode, level 160/161/165/163"
Do you know what these values ​​mean? Or is there any way to recognize the power as shown in the guide? 
Thanks again!

**Reply by T-Troll** [reply]
It's exactly the same as a "Thermal mode" into AWCC.
Unfortunately, mode names can't be retrieved and vary from system to system, so i use BIOS codes instead (160...etc). But you can edit it yourself into edit box.
Use HWINFO/some GPU test to get a hint - all of them have different CPU/GPU PL settings, as well as fan settings.

**Reply by Und3r22** [reply]
I imagined it should be like the AWCC.
I had already done a quick test, but I didn't monitor everything so I wasn't sure, as soon as I have time I will test it and post my results to help anyone who has the same system.
Thanks again.
